Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,900 @@
+      - "73"
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: alert.name
+            iscontext: true
+          right:
+            value:
+              simple: Suspicious API call from a Tor exit node
+        - operator: isEqualNumber
+          left:
+            value:
+              complex:
+                root: DBotScore
+                accessor: Score
+            iscontext: true
+          right:
+            value:
+              simple: "3"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 4f102212-5337-40d7-8477-f983500618fe
+    type: regular
+    task:
+      id: 4f102212-5337-40d7-8477-f983500618fe
+      version: -1
+      name: Load alert JSON
+      description: Loads a json from string input, and returns a json object result
+      scriptName: LoadJSON
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "68"
+    scriptarguments:
+      extend-context:
+        simple: alertJson=
+      ignore-outputs:
+        simple: "true"
+      input:
+        complex:
+          root: alertData.alerts
+          accessor: original_alert_json
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "68":
+    id: "68"
+    taskid: 3d769696-08fc-4298-880f-a47793a9a3f2
+    type: title
+    task:
+      id: 3d769696-08fc-4298-880f-a47793a9a3f2
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "80"
+      - "82"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "73":
+    id: "73"
+    taskid: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+    type: regular
+    task:
+      id: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+      version: -1
+      name: Investigate collected data
+      description: |2-
+         To determine if this is a true positive event,  review the operations performed by the access key and the user in the recent time frame.
```

Comment:
Fix the syntax to be presented in a comfort way for reading using markdown.

Reply:
Done

Reply:
<img width="422" alt="image" src="https://user-images.githubusercontent.com/28757135/225636555-b31300ee-8dcd-4afb-bf1e-f48a8405963b.png">
Doesn't look so good.